packages/react-heartwood-components/src/components/View/View.js

```diff
@@ -11,79 +11,60 @@ type Props = {
 	business: Object,
 	getSearchSuggestionValue?: Function,
 	renderSearchSuggestion?: Function,
-	children: Node
+	children: Node,
+	setSidebarVisibility: Function,
+	setSidebarExpanded: Function,
+	forceCloseSidebar: Function,
+	isSidebarVisible?: boolean,
+	isSidebarExpanded?: boolean
 }
-type State = {
-	sidebarIsVisible: boolean,
-	sidebarIsExpanded: boolean
-}
-
-export default class View extends Component<Props, State> {
-	state = {
-		sidebarIsVisible: false,
-		sidebarIsExpanded: true
-	}
-
-	toggleSidebarVisibility = () => {
-		this.setState(prevState => ({
-			sidebarIsVisible: !prevState.sidebarIsVisible,
-			sidebarIsExpanded: true
-		}))
-	}
 
-	toggleSidebarExpanded = () => {
-		this.setState(prevState => ({
-			sidebarIsExpanded: !prevState.sidebarIsExpanded
-		}))
-	}
+const View = (props: Props) => {
+	const {
+		sidebarItems,
+		user,
+		business,
+		getSearchSuggestionValue,
+		renderSearchSuggestion,
+		isSidebarVisible,
+		isSidebarExpanded,
+		setSidebarExpanded,
```

Comment:
Ah so. I think in react-heartwood this prop should actually be `onToggleSidebarExpanded`, since from this component's perspective it's a simple callback to alert downstream components that the user has requested the bool to flip. In other words, at this level it's not "setting" anything.

This is more in-line with typical controlled components, like forms, where you get an `onChange` prop callback and are responsible for updating your state to present the updated prop back to the controlled component